src/components/pages/Offers/Offers.jsx

```diff
@@ -400,6 +426,7 @@ class Offers extends PureComponent {
                 <tbody className="offers-list">
                   {offers.map(offer => (
                     <OfferItemContainer
+                      areAllOffersSelected={areAllOffersSelected}
                       isSelected={selectedOfferIds.includes(offer.id)}
```

Comment:
```suggestion
isSelected={areAllOffersSelected || selectedOfferIds.includes(offer.id)}
```
Mais ca n'est surment pas necessaire car tu remplis bien `selectedOfferIds` avec toutes les id de la page dans `selectAllOffers`